src/main/scala/tech/cryptonomic/conseil/generic/chain/DataTypes.scala

```diff
@@ -55,11 +56,9 @@ object DataTypes {
   }
 
   /** Method formatting millis to ISO format */
-  def formatToIso(epochMillis: Long): String =
-    Instant.ofEpochMilli(epochMillis)
-      .atZone(ZoneOffset.UTC)
-      .format(ISO_INSTANT)
-
+  def formatToIso(epochMillis: Long): String = {
+    DateTimeFormatter.ISO_INSTANT.format(new Timestamp(epochMillis).toInstant)
```

Comment:
I can't figure out why passing through Timestamp should change things, wrt. creating the Instant.

Did you see a difference?

Reply:
I think I got to the bottom of my issue and the solution was to use just `new Timestamp(_).toString`: 
- database does not care about the time zone
- instant does
So what problems did I have there - pick 2 out of three - Travis, tests, running locally 😄. Two of them worked and third not when I tried to configure it with `Instant`. `Timestamp` is a generic solution.

Reply:
so, we assume that
- the DB will always receive timestamps from the TZ that we choose to use to create the value
- it returns the timestamp as-is, with no notion of TZ
- if we read an Instant from the Timestamp, then something bad happens?

Reply:
First of all, we don't have much control on what we will receive from DB.
Next if we read Instant from the Timestamp and try to use it against DB we will get incorrect results. Instant would need to be used with TZ which changes - best example is when you try running tests locally and on Travis. 
We are relying on the consistent usage of timestamp across Conseil and Lorre. Maybe it would be better to have `Timestamp with timezone` in the DB. But it would require using Instant everywhere.

Reply:
What I mean is that consistency between different environment should be guaranteed exactly by using timezones correctly.
When Lorre stores a timestamp in DB, it should be expressed relative to a standard timezone (UTC being the most common one).
Hence every time we read timestamps from there we should know that they need be expressed as epoch relative to that timezone.
It's reasonable that running a query from a different timezone returns different results, unless we normalise the query input too...

If you're in Warsaw and ask for 14.00-15.00, would you expect to have data timestamped on db with 14.00-15.00 or 12.00-13.00 ?

Reply:
But we are not using timezones explicitly anywhere, so I don't see a reason to use it here. I agree with your argument that we probably should, that's why I mentioned changing DB columns to timestamp with timezone. My understanding is that frontend will be handling time zones thats why Mike told on Riot that he expects timestamp in millis.

Reply:
ok, I agree that, as long as the client knows what it's asking for (in terms of date-strings) we're cool